http/Cargo.toml

```diff
@@ -15,15 +15,15 @@ repository = "https://github.com/twilight-rs/twilight.git"
 version = "0.6.1"
 
 [dependencies]
-rand = { default-features = false, features = ["std_rng", "std"], version = "0.8" }
-hyper = { default-features = false, features = ["client", "http1", "http2", "runtime"], version = "0.14" }
+hyper = { default-features = false, features = ["client", "http1", "http2"], version = "0.14" }
 hyper-rustls = { default-features = false, optional = true, version = "0.22" }
 hyper-tls = { default-features = false, optional = true, version = "0.5" }
 percent-encoding = { default-features = false, version = "2" }
-tokio = { default-features = false, features = ["time"], version = "1.0" }
-twilight-model = { default-features = false, path = "../model" }
+rand = { default-features = false, features = ["std_rng", "std"], version = "0.8" }
 serde = { default-features = false, features = ["derive"], version = "1" }
 serde_json = { default-features = false, features = ["std"], version = "1" }
+tokio = { default-features = false, features = ["sync", "time"], version = "1.0" }
```

Comment:
Why was `sync` enabled?

Reply:
It's used for ratelimiting (`src/ratelimiting/{bucket,mod}.rs`)